src/geometry/collider.rs

```diff
@@ -570,3 +606,11 @@ impl ColliderBuilder {
         }
     }
 }
+
+fn volume(shape: &SharedShape) -> Real {
+    mass(&shape.mass_properties(1.0)) // TODO: add SharedShape::volume to parry
+}
+
+fn mass(mp: &MassProperties) -> Real {
+    crate::utils::inv(mp.inv_mass) // TODO: add MassProperties::mass() to parry
+}
```

Comment:
Switching to `Either` will make this unnecessary. Though I agree it would be nice to add to Parry what the `TODO`s are suggesting.